.github/workflows/build_and_run.yml

```diff
@@ -48,7 +48,7 @@ jobs:
           conda install scipy scikit-learn pybind11
           conda install -c pkgs/main libgcc-ng">=11.2.0" libstdcxx-ng">=11.2.0" libgomp">=11.2.0"
           conda install -c dppy/label/dev -c intel -c main dpctl numba-dpex dpnp
-          pip install alembic
+          pip install alembic tomli
```

Comment:
Does the README need updating?

Reply:
Nice catch! Added. Also install tomli via conda now